draft-ietf-quic-transport.md

```diff
@@ -302,7 +302,7 @@ deployed and used concurrently. Version negotiation is described in
 
 QUIC versions are identified using a 32-bit unsigned number.
 
-The version 0x00000000 is reserved to represent an invalid version.  This
+The version 0x00000000 is reserved to represent version negotiation.  This
 version of the specification is identified by the number 0x00000001.
```

Comment:
OT Q: Why isn't version 0 represented by the number 0x00000000?

Reply:
I'm a little confused by the question.  Are you asking why it's spelled out?

Reply:
I misread these two sentences as "The version 0x00000000 is reserved to represent an invalid version.  This(Ian's misinterpretation: the version negotiation version) version of the specification is identified by the number 0x00000001."

So I misinterpreted the use of the word "This", which is partly a product of the small snippet Github displays, but possibly it makes sense to clarify this now that the spec uses two versions?